.github/workflows/deployment.yaml

```diff
@@ -30,3 +30,11 @@ jobs:
             npm i
           fi
           yarn deploy
+      - name: Generate Seneca-IPC144.pdf
+        run: |
+          chmod 777 ./convert-to-pdf.sh
```

Comment:
This shouldn't be necessary (i.e., it should be executable in git):

```
chmod +x ./convert-to-pdf.sh
git add convert-to-pdf.sh
```

And commit that.  Git stores file attributes as well as contents.